src/runtime/experiment-flags.ts

```diff
@@ -36,4 +36,9 @@ export enum ExperimentFlags {
    * Experiment flag to enable populating the client config in the Classic runtime.
    */
   POPULATE_CLIENT_CONFIG_CLASSIC = 'populate-client-config-classic',
+
+  /**
+   * Experiment for allowing rewarded ads to be rendered.
+   */
+  REWARDED_ADS_ENABLED = 'rewarded_ads_enabled',
```

Comment:
Can this be controlled by a backend flag instead?

Reply:
If we do need a client experiment, see the note on L20 about adding the flag to test configurations.

Reply:
We can filter out the action in the article endpoint, but I figured it would be better to maintain the logic in one place. Is there a reason we don't want the flag in the client?

Reply:
Client flags are generally more difficult to work with. They aren't integrated as well with upstream systems and they require binary pushes to change. I also find it cleaner to implement the business logic in the server where possible, instead of sending data the client may ignore. In this case I think using a server-side flag would still result in maintaining the logic in a single location.

Reply:
Got it, I'll keep the flag out.